src/services/http_service.ts

```diff
@@ -32,7 +36,10 @@ export class HttpService {
    *     False if the request doesn't accept any of the passed in types,
    *     or the content type that was matches
    */
-  public accepts(request: any, type: string | string[]): boolean | string {
+  public accepts(
```

Comment:
Has the doc block been updated to reflect this?

Reply:
c6c3084 and b73933a